src/pages/Integrations/Create/CreateWizard.tsx

```diff
@@ -19,15 +20,15 @@ export const CreateWizard: React.FunctionComponent<CreateWizardProps> = ({
 }: CreateWizardProps) => {
   const mapperExtension = {
     'summary-content': Review,
+    'card-select': CardSelect
```

Comment:
can you please make the names constants? Something like...

```suggestion
    [SUMMARY]: Review,
    [CARD_SELECT]: CardSelect
```